spec/controllers/control_panel/boards_controller_spec.rb

```diff
@@ -148,7 +220,7 @@
 
       it "flashes user auth warning message" do
         get :index
-        expect(flash[:alert]).to eq I18n.t('user.auth.failure')
+        expect(flash[:alert]).presence
```

Comment:
This is good practise to use .presence when you don't have I18n.translations but in this situation I would prefer check the translation key because you will check 2 thinks:
1. alert presence
2. If it is correct message.